src/generator/UNetGANGenerator.py

```diff
@@ -0,0 +1,55 @@
+from src.generator.Generator import Generator
+import numpy as np
+from src.environment.EnvironmentManager import EnvironmentManager as em
+
+
+class UNetGANGenerator(Generator):
+    _DIM_Z = 128
+    _PATH_TO_IMAGES = "environment/unetgan/out/sample/"
+    _IMAGE_EXT = ".png"
+
+    def __init__(self):
+        super().__init__("U-NetGAN")
+
+    def latent_space_std(self) -> np.ndarray:
```

Comment:
You may not need to re-doc the function if the superclass has the same docstring, and if nothing more/new needs to be described. It does not hurt to do this, but it may be redundant (if I'm not mistaken, check before skipping if the docstring propagates properly)